### ISSUE TITLE
[BUG] PHP compile errors generated when WordPress is using a custom content directory

### ISSUE THREAD

**Issue by dd32** [codefoxes/greenlet#52]
**Describe the bug**
When WordPress is installed in a subdirectory and/or has a custom content directory, the greenlet theme fails to detect the file path and hits a compile error situation.

For example, this is the error generated on wp-themes.com for every demo view of the theme:
```
E_COMPILE_ERROR: wp-content/themes/greenlet/library/common/helpers.php:175 - require_once(): Failed opening required 'http://wp-themes.com/wp-content/themes/greenlet/assets/js/scripts.min.js' (include_path='.:')
```

One way to trigger this is to have WordPress installed with a custom content directory like this:
```
/wp/ <-- wordpress files
/wp-content/themes/ <-- Theme files
/wp-config.php <-- Config file, with define( 'WP_CONTENT_DIR', __DIR__ . '/wp-content/' );
```

This is ultimately happening as `$path = str_replace( get_home_url(), ABSPATH, $src );` is not a good way to convert a URL to a file path. If you must do it, you could probably use something like the following to also take into account a custom content directory.
```
$path = str_replace(
   array(
      content_url(),
      home_url(),
   ),
   array(
      WP_CONTENT_DIR,
      ABSPATH,
   ),
   $src
);
```

Further more, [if you could replace the](https://github.com/codefoxes/greenlet/blob/master/library/common/helpers.php#L172-L177) `ob_start(); require $path; ob_get_clean()` and replace it with `file_get_contents()` that'll also avoid some issues around this.
Yes, WordPress.org Theme Check does complain about it currently, but it's allowed and the warning will be removed shortly, there's no need to work around it with output buffering.

**Comment by karthikax** [comment]
Thanks a lot for reporting the issue and for possible solutions @dd32 
We are able to reproduce the issue. We should have a fix soon

**Comment by karthikax** [comment]
We have added content_url replacement for enqueue URL to path conversion and fixed the issue in the latest version: 2.2.0.
We are also looking for a better permanent solution. Also, w.org is not allowing multiple warnings at the moment. Will convert object buffering to file_get_contents at the earliest. Thanks for your contribution.
